IBAnimatable/ActivityIndicatorAnimationBallClipRotateMultiple.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  Created by Tom Baranes on 23/08/16.
+//  Copyright (c) 2016 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+public class ActivityIndicatorAnimationBallClipRotateMultiple: ActivityIndicatorAnimating {
+
+  // MARK: Properties
+
+  private let duration: CFTimeInterval = 0.7
+
+  // MARK: ActivityIndicatorAnimating
+
+  public func configAnimation(in layer: CALayer, size: CGSize, color: UIColor) {
+    let bigCircleSize: CGFloat = size.width
+    let smallCircleSize: CGFloat = size.width / 2
+    let longDuration: CFTimeInterval = 1
+    let timingFunction = CAMediaTimingFunction(name: kCAMediaTimingFunctionEaseInEaseOut)
+    circleOf(shape: .RingTwoHalfHorizontal,
+             duration: longDuration,
+             timingFunction: timingFunction,
+             layer: layer,
+             size: bigCircleSize,
+             color: color, reverse: false)
+    circleOf(shape: .RingTwoHalfVertical,
+             duration: longDuration,
+             timingFunction: timingFunction,
+             layer: layer,
+             size: smallCircleSize,
+             color: color, reverse: true)
+  }
+
+}
+
+// MARK: - Setup
+
+private extension ActivityIndicatorAnimationBallClipRotateMultiple {
+
+  func createAnimationIn(duration duration: CFTimeInterval, timingFunction: CAMediaTimingFunction, reverse: Bool) -> CAAnimation {
+    // Scale animation
+    let scaleAnimation = CAKeyframeAnimation(keyPath: "transform.scale")
+    scaleAnimation.keyTimes = [0, 0.5, 1]
+    scaleAnimation.timingFunctions = [timingFunction, timingFunction]
```

Comment:
I prefer let this one like this for now since it's just a port of NVActivityIndicatorView